templates/prusa-mk3s-printer.template.cfg

```diff
@@ -0,0 +1,229 @@
+# This file contains pin mappings for Prusa i3 MK3S+ printers. To use
+# this config, the firmware should be compiled for the AVR atmega2560.
+
+# See docs/Config_Reference.md for a description of parameters.
+
+
+#############################################################################################################
+### CONTROL BOARD
+### Pick the board you have installed and wired in your printer.
+#############################################################################################################
+[include RatOS/boards/prusa-einsy/config.cfg]
+#[include RatOS/boards/btt-skr-2-429/config.cfg]
+#[include RatOS/boards/btt-octopus-11/config.cfg]
+#[include RatOS/boards/btt-octopus-11-407/config.cfg]
+#[include RatOS/boards/btt-octopus-pro-429/config.cfg]
+#[include RatOS/boards/btt-octopus-pro-446/config.cfg]
+#[include RatOS/boards/btt-skr-pro-12/config.cfg]
+#[include RatOS/boards/fysetc-spider/config.cfg]
+
+### RPI Mcu (Necessary for dual sensor input shaping)
+#[include RatOS/boards/rpi/config.cfg]
+
+#############################################################################################################
+### BASE SETUP
+#############################################################################################################
+[include RatOS/printers/prusa-mk3s/prusa-mk3s.cfg]
+
+#############################################################################################################
+### STEPPER MOTORS, DRIVERS & SPEED LIMITS
+### Pick the drivers and stepper motors you're using. See the RatOS documentation for custom combinations.
+#############################################################################################################
+[include RatOS/printers/prusa-mk3s/steppers.cfg]
+
+# UNCOOLED TMC 2209 + LDO-42STH40-1684AC
+# It is recommended that you enable these settings until you have verified that your printer moves as expected
+#[include RatOS/printers/prusa-mini/tmc2209.cfg]
+[include RatOS/printers/prusa-mk3s/speed-limits-basic.cfg]
+[include RatOS/steppers/ldo/42sth40-1684ac/2130/24v-0.4a-*.cfg]
+
+# COOLED TMC 2209 + LDO-42STH40-1684AC
+# It is highly recommended that you enable these settings as soon as you have verified that your printer moves 
+# as expected. Actively cooled drivers are needed (ie, such as the stock prusa-mini eletronics enclosure with 
+# the fan connected)
+# These settings increase torque, precision and speed, and makes sure your motors perform to their maximum
+# potential.
+#[include RatOS/printers/prusa-mini/tmc2209.cfg]
+#[include RatOS/printers/prusa-mini/speed-limits-performance.cfg]
+#[include RatOS/steppers/ldo/42sth40-1684ac/2209/24v-1.188a-*.cfg]
+
+# STEALTH MODE
+# NOTE: You still need to include one of the above stepper motor definitions.
+# NOTE: This will make your printer quiter but less accurate, it's an inherent side effect of stealthchop.
+[include RatOS/printers/prusa-mk3s/tmc2130-stealth.cfg]
```

Comment:
This does not actually run stealthchop.

Reply:
Solved, I have created a separate file in order to configure the stealthchop, and also I corrected the values